src/test/java/com/flowingcode/vaadin/addons/orgchart/OrgchartDemoView.java

```diff
@@ -58,19 +59,19 @@ public OrgchartDemoView() {
 			this.removeAll();
 			switch (e.getSelectedTab().getLabel()) {
 			case DRAGNDROP_DEMO:
-				iframe.getElement().setAttribute("srcdoc", getSrcdoc(DRAGNDROP_DEMO));
```

Comment:
Commit this change ("Improved getSrcdoc method") separately. May be in the same branch, but as a different commit.